conda/cli/install.py

```diff
@@ -50,6 +50,10 @@
 
 
 def check_prefix(prefix, json=False):
+    if os.pathsep in context.target_prefix:
```

Comment:
In this case the prefix is passed in:

```suggestion
    if os.pathsep in prefix:
```

Reply:
As long as prefix includes the whole installation path.

Reply:
`check_prefix` is used in two places, in one case it passes in `context.target_prefix`, in the other it effectively passes it in:

https://github.com/conda/conda/blob/5c814f97ac02af80477eaf89229a7c2b66814ca0/conda/cli/install.py#L141-L143

https://github.com/conda/conda/blob/5c814f97ac02af80477eaf89229a7c2b66814ca0/conda_env/cli/main_create.py#L118-L122